src/rules/maxFileLineCountRule.ts

```diff
@@ -55,17 +54,13 @@ export class Rule extends Lint.Rules.AbstractRule {
     }
 
     public apply(sourceFile: ts.SourceFile): Lint.RuleFailure[] {
-        const ruleFailures: Lint.RuleFailure[] = [];
-        const ruleArguments = this.getOptions().ruleArguments;
-        const lineLimit: number = ruleArguments[0];
+        const lineLimit: number = this.ruleArguments[0];
         const lineCount: number = sourceFile.getLineStarts().length;
-        const disabledIntervals = this.getOptions().disabledIntervals;
-
-        if (lineCount > lineLimit && disabledIntervals.length === 0) {
```

Comment:
This changes the behavior of the rule, which should at least be documented.
Before this change the rule would not fail if there were **any** disabledInvervals. Now you would need to have a disable comment in the first line and first column to disable the rule.

You would only get back the old behavior if the failure would span the whole file. I would not suggest doing that.